connectors/sources/sharepoint.py

```diff
@@ -36,7 +36,10 @@
 RETRIES = 3
 FILE_SIZE_LIMIT = 10485760
 CHUNK_SIZE = 1024
-TOP = 5000
+if "RUNNING_FTEST" in os.environ:
```

Comment:
This is not needed anymore. Could you please remove it?

Reply:
We have added this condition to decrease the pagination size.. otherwise it will return 5000 response for one api call and if those 5000 response process further for lists so it will return huge number of records which takes more than 10minutes and limit will exceed for ftest in that case.

Reply:
I'm not sure I get this. 

There was a configuration `start_lists, end_lists = 0, 450` that generated about 5k docs in total. If I requests TOP=10k but there are only 1k of docs shouldn't the server return only 1k docs?